src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser.md

```diff
@@ -11,53 +11,81 @@ Note that because k6 does not run in NodeJS, the browser module APIs will slight
 
 <Blockquote mod="note" title="">
 
-To work with the browser module, make sure you are using [k6 version 0.43.0](https://github.com/grafana/k6/releases/tag/v0.43.0) or above.
+To work with the browser module, make sure you are using the latest [k6 version](https://github.com/grafana/k6/releases).
 
 </Blockquote>
 
-## Modules
+## Importable Properties
 
 The table below lists the importable properties from the top level module (`'k6/experimental/browser'`).
 
 | Property | Description                                                                                                                                                                          |
 |----------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| chromium | A [BrowserType](/javascript-api/k6-experimental/browser/browsertype) to launch tests in a Chromium-based browser.                                                                                |
+| browser | The browser module API is the entry point for all your tests. See the [example](#example) and the [API](#browser-module-api) below. |
 | devices  | Returns predefined emulation settings for many end-user devices that can be used to simulate browser behavior on a mobile device. See the [devices example](#devices-example) below. |
-| version  | Returns the version number of k6 browser.                                                                                                                                           |
 
-### Devices Example
 
-To emulate the browser behaviour on a mobile device and approximately measure the browser performance, you can import `devices` from `k6/experimental/browser`.
+## Browser Module API
 
-<CodeGroup labels={["script.js"]} lineNumbers={[true]}>
+The browser module is the entry point for all your tests, and it is what interacts with the actual web browser via [Chrome DevTools Protocol](https://chromedevtools.github.io/devtools-protocol/) (CDP). It manages:
+- [BrowserContext](/javascript-api/k6-experimental/browser/browsercontext/) which is where you can set a variety of attributes to control the behavior of pages;
+- and [Page](/javascript-api/k6-experimental/browser/page/) which is where your rendered site is displayed.
+
+A new Chromium browser process is automatically launched when you use the `newContext` method of the `browser` module (`'k6/experimental/browser'`).
```

Comment:
I don't think we need to mention this, as it seems more like a technical detail. Also, this will most probably not be true once we integrate with k6 event system, in which situation, the browser process will be initialized when the iteration starts, if the browser type is configured.

Reply:
👍  See: f544e0e0